src/ImageSharp/Formats/Pbm/PlainDecoder.cs

```diff
@@ -65,20 +65,30 @@ private static void ProcessGrayscale<TPixel>(Configuration configuration, Buffer
         using IMemoryOwner<L8> row = allocator.Allocate<L8>(width);
         Span<L8> rowSpan = row.GetSpan();
 
+        bool eofReached = false;
         for (int y = 0; y < height; y++)
         {
             for (int x = 0; x < width; x++)
             {
-                byte value = (byte)stream.ReadDecimal();
-                stream.SkipWhitespaceAndComments();
-                rowSpan[x] = new L8(value);
+                stream.ReadDecimal(out int value);
```

Comment:
Shouldn't we test here?

Reply:
I guess it doesn't matter since we're simply assigning 0.

Reply:
No, when it's the last digit of the file the value is valid! For simplicity, I'm ignoring the retval here and letting the `SkipWhitespaceAndComments` call below to detect EOF.